server/src/test/java/io/spine/examples/chatspn/server/e2e/TestUser.java

```diff
@@ -0,0 +1,200 @@
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.examples.chatspn.server.e2e;
+
+import com.google.common.collect.ImmutableList;
+import io.grpc.ManagedChannel;
+import io.grpc.StatusRuntimeException;
+import io.spine.base.CommandMessage;
+import io.spine.base.EntityColumn;
+import io.spine.client.Client;
+import io.spine.core.UserId;
+import io.spine.examples.chatspn.ChatId;
+import io.spine.examples.chatspn.account.UserChats;
+import io.spine.examples.chatspn.account.UserProfile;
+import io.spine.examples.chatspn.account.command.CreateAccount;
+import io.spine.examples.chatspn.chat.ChatPreview;
+import io.spine.examples.chatspn.chat.command.CreatePersonalChat;
+import io.spine.examples.chatspn.chat.command.DeleteChat;
+import io.spine.examples.chatspn.message.MessageView;
+import io.spine.examples.chatspn.message.command.EditMessage;
+import io.spine.examples.chatspn.message.command.RemoveMessage;
+import io.spine.examples.chatspn.message.command.SendMessage;
+import io.spine.net.EmailAddress;
+
+import static io.grpc.ManagedChannelBuilder.forAddress;
+import static io.grpc.Status.CANCELLED;
+import static io.spine.client.Client.usingChannel;
+import static io.spine.client.OrderBy.Direction.ASCENDING;
+import static io.spine.client.QueryFilter.eq;
+import static io.spine.examples.chatspn.server.e2e.ServerRunningTest.port;
+import static io.spine.examples.chatspn.server.e2e.given.TestUserEnv.chatPreview;
+import static io.spine.examples.chatspn.server.e2e.given.TestUserEnv.createAccount;
+import static io.spine.examples.chatspn.server.e2e.given.TestUserEnv.createPersonalChat;
+import static io.spine.examples.chatspn.server.e2e.given.TestUserEnv.deleteChatCommand;
+import static io.spine.examples.chatspn.server.e2e.given.TestUserEnv.editMessageCommand;
+import static io.spine.examples.chatspn.server.e2e.given.TestUserEnv.messageView;
+import static io.spine.examples.chatspn.server.e2e.given.TestUserEnv.removeMessageCommand;
+import static io.spine.examples.chatspn.server.e2e.given.TestUserEnv.sendMessage;
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.junit.jupiter.api.Assertions.fail;
+
+/**
+ * Registered user with own {@link Client}
+ * and API to send commands and read projections.
```

Comment:
Let's not explain the API in the documentation. Otherwise we'll have to update the class-level documentation every time something changes.